### ISSUE TITLE
Manual update

### ISSUE THREAD

**Issue by Manocophil** [36bits/msmquote#3]
After updating the quotes with msmquote, if I want to manually update a quote in Money I have to press several times the refresh button (more than 50) to update the quote. This behaviour oocurs only for the first manually update in Money.
Money version: 14.0.120.2107
Msmquote version: 3.1.3

**Comment by 36bits** [comment]
Hi, please could you let me know the symbol of the investment that is causing this issue. Thanks.

**Comment by Manocophil** [comment]
Hi,
It's not for a specific symbol. It happens for any investment after a msmquote update
Regards

**Comment by 36bits** [comment]
Hi, I've tried to reproduce the issue that you're experiencing but with no success. It looks like you're using the French edition of Money 2005: I've created a test portfolio in this edition and added two stocks. I've then run msmquote which updates the quote data as expected. After that I manually changed the price of the stocks and this was instantly reflected in the portfolio page without having to press refresh.

Maybe try running a 'rapid repair' from the file menu and see if that makes a difference.

Regards.
